src/pages/home/AppWrapper.js

```diff
@@ -41,7 +29,7 @@ const defaultProps = {
     network: {isOffline: true},
 };
 
-class HomePage extends PureComponent {
+class AppWrapper extends PureComponent {
```

Comment:
is this component needed anymore? can prob move the business logic up a component